core/src/test/scala/examples/DummyProducer.scala

```diff
@@ -1,10 +1,17 @@
-package com.softwaremill.react.kafka2
+package examples
 
 import akka.actor.ActorSystem
 import akka.stream.scaladsl.Source
 import akka.stream.{ActorMaterializer, ActorMaterializerSettings}
 import org.apache.kafka.common.serialization.{ByteArraySerializer, StringSerializer}
 
+import com.softwaremill.react.kafka2._
+
+// tbd. What is the purpose of this?
```

Comment:
Publish `1 to 10000` into kafka and waiting for publishing confirmation. Also provides an example of correct shutdown